redbot/cogs/downloader/downloader.py

```diff
@@ -1735,7 +1735,7 @@ async def findcog(self, ctx: commands.Context, command_name: str) -> None:
                     if cog_installable.repo is None
                     else cog_installable.repo.name
                 )
-                cog_name = cog_installable.name
+                cog_name = "{} ({})".format(cog_installable.name, cog.__class__.__name__)
```

Comment:
```suggestion
                cog_name = f"{cog_installable.name} ({cog.__class__.__name__})"
```
f-string are faster and more modern (See #5427 for tests result), they should be preferred. 😄

Reply:
welp in my testing i initially used f-strings, but a few days ago i was told that they are a bit slower & and mess up translation so hence i didn't use them. Thanks for the quick response and correction though.